src/allmydata/test/test_crypto.py

```diff
@@ -0,0 +1,449 @@
+import six
+import unittest
+
+from base64 import b64decode
+from binascii import a2b_hex, b2a_hex
+from os import path
+
+from allmydata.crypto import (
+    aes,
+    ed25519,
+    rsa,
+    remove_prefix,
+    BadPrefixError
+)
+
+
+RESOURCE_DIR = path.join(path.dirname(__file__), 'data')
```

Comment:
`twisted.python.filepath` is really quite nice

Reply:
hmm, yes it is :)